src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2872,21 +2871,31 @@ namespace WaterToAirHeatPumpSimple {
                          .glycol->getSpecificHeat(state, state.dataSize->PlantSizData(PltSizNum).ExitTemp, RoutineNameAlt);
 
                 if (simpleWatertoAirHP.WAHPType == WatertoAirHP::Heating) {
-                    RatedWaterVolFlowRateDes = simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
+                    RatedWaterVolFlowRateDes =
+                        (1 + 1 / RatedHeatCOP) * simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
```

Comment:
Here we're dealing with the heating coil so the evaporator is on the source side and the condenser is on the load side.

So if we start with `Qc = Qe + P` and because we're dealing with heating operation `COP = Qc / P` so `Qe = Source Side Load = Qc - P = Qc (1 - 1 / COP)` and `Qc = simpleWatertoAirHP.RatedCapHeat`. So I think that the `+` sign should be changed to `-`.

Instead of:
```
RatedWaterVolFlowRateDes =
(1 + 1 / RatedHeatCOP) * simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
```
We should have:
```
RatedWaterVolFlowRateDes =
(1 - 1 / RatedHeatCOP) * simpleWatertoAirHP.RatedCapHeat / (state.dataSize->PlantSizData(PltSizNum).DeltaT * Cp * rho);
```

Reply:
I agree with the suggested change. See the equation below for reference: 

The source side (evaporator coil) fluid mass flow rate can be calculated from the load side heating capacity and from the heating mode COP_h as follows:

`m_dot_source = Q_load_side * (1 - 1/cop_h) / (cp_source * deltaT_source)`

Alternatively: 

`m_dot_source = Q_source_side * / (cp_source * deltaT_source)`

Where,

`Q_source_side = Q_load_side * (1 - 1/COP_h)`